src/fonts/OTF/TeX/makeFF

```diff
@@ -253,10 +254,6 @@ $map{cmsy10} = {
     0x7F => 0x2660,         # \spadesuit
   ],
 
-  "Math-Italic" => [
-    0x36 => 0x2F            # \not
-  ],
-
```

Comment:
As far as I can tell, these are not used at all in KaTeX (they seem out of date and wrong). But if it would be less controversial to leave them in, I can put them back.

Reply:
To investigate this further (given that `0x2F` is the forward slash, `/`), I tried this test:

```latex
x/y\mathit{/}z\mathbf{/}w\mathrm{/}x\mathnormal{/}y\mathsf{/}z\mathtt{/}w\bm{/}x
```

Here's old KaTeX's rendering:

![image](https://user-images.githubusercontent.com/2218736/53022419-23c61b80-3429-11e9-90f7-bd050cf58c22.png)

Here's the rendering with this PR applied:

![image](https://user-images.githubusercontent.com/2218736/53022446-304a7400-3429-11e9-89f1-00ca83c05094.png)

So this PR fixes the behavior of `\mathnormal{/}` but `\mathit{/}` still seems broken (it seems to render a `\not` symbol incorrectly).  What's weird is that both bare `/` and `\mathit{/}` render in `font-family: KaTeX_Main`; the latter differs only in a `font-style: italic` spec. Not sure what's going on here... (though it was also a bug before this PR)

Reply:
Ah, the problem is the huge `advanceWidth` on the `/` (0x2f) glyph in the Main-Regular font (which I [previously spotted here](https://github.com/KaTeX/KaTeX/issues/1842#issuecomment-461632562) and thought was relevant to `\not`, though it isn't):

![image](https://user-images.githubusercontent.com/2218736/53023203-bfa45700-342a-11e9-82d1-fb54b193943e.png)

I'm still not sure where this comes from...  As far as I can tell, this glyph is from these rules in `makeFF`:

```perl
$map{cmti10} = {
  "Main-Italic" => [
    ...
    [0x25,0x2F] => 0x25,    # %, &, ', (, ), *, +, comma, -, ., /
```

All the other characters in that range render fine.

Reply:
I found the bug: some old `$extra` code:
```perl
  not => [
    'Select(0u2F)',
    'SetRBearing(778,1)',
  ],
```

The example above now renders correctly!

![image](https://user-images.githubusercontent.com/2218736/53023880-0a729e80-342c-11e9-9049-74797208f2f8.png)